### DISCUSSION TITLE
question on using just your containers

### DISCUSSION THREAD

**Discussion by ehmueller** [telekom-security/tpotce#922] [Q&A]
Hello,

I am not sure if this is the correct place to ask my question, however this was the only method I could find that will allow me to ask my question.

I am running a Ubuntu 20.04 system. Can I just run the containers of my choice on my system and then send the container data to my own elastic environment?

thanks
Eric

**Comment by t3chn0m4g3** [comment]
T-Pot is tailored around Debian and I doubt it will work correctly out of the box if you choose post install on Ubuntu (which you need to modify the scripts for).
If you choose to install on Debian all you need is to forward the logs to your central solution, i.e. with Filebeat.
